utils/index.sh

```diff
@@ -15,10 +15,13 @@ setup(){
         export KUBECONFIG=/home/airflow/auth/config
         curl -sS https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xz oc
         export PATH=$PATH:/home/airflow/.local/bin:$(pwd)
-    else [[ -z $AIRFLOW_CTX_DAG_ID ]]
+    elif [[ -n $PROW_JOB_ID ]]; then
         export ci="PROW"
         export prow_base_url="https://prow.ci.openshift.org/view/gs/origin-ci-test/logs"
         export RELEASE_STREAM=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '-' -f1-2) || echo "Cluster Install Failed"
+    elif [[ -n $BUILD_ID ]]; then 
+        export ci="JENKINS"
+        export build_url=${BUILD_URL}
```

Comment:
I think we might want to add `RELEASE_STREAM` value as here, as it is not being populated in the doc as of now.

Reply:
For jenkins I dont know how we would set that variable other than the ocp version, should I duplicate that value for consistency?

Reply:
should we set this value the same way for all 3 ci types? like set it before/after the if statements?

Reply:
We need not necessarily duplicate it, but have it set for non AIRFLOW runs, i.e PROW and JENKINS.